v3.6/security/kubernetes-network-policy.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Kubernetes network policy
+---
+
+### Big picture
+
+Kubernetes network policy lets administrators and developers enforce network connections using rules. 
+
+### Value
+
+Kubernetes network policy lets developers secure access to and from their applications using the same simple language they use to deploy them. Developers can focus on their applications without understanding low-level networking concepts. Enabling developers to easily secure their applications using network policies supports a shift left DevOps environment. 
+
+### Features
+
+The Kubernetes Network Policy API supports the following features: 
+
+- Policies are namespace scoped
+- Policies are applied to pods using label selectors
+- Policy rules can specify the connections that are allowed to/from pods, namespaces, or CIDRs
+- Policy rules can specify protocols (TCP, UDP, SCTP), named ports or port numbers
+
+### Concepts
+
+The Kubernetes Network Policy API provides a standard way for users to define network policy for controlling network connections. However, Kubernetes has no built-in capability to enforce the network policy. To enforce network policy, you must use a network plugin such as Calico. 
+
+#### Ingress and egress
+
+The bulk of securing network connections typically revolves around defining egress and ingress rules. From the point of view of a Kubernetes pod, **ingress** is incoming connections to the pod, and **egress** is outgoing connections from the pod. In Kubernetes network policy, you create ingress and egress “allow” rules independently (egress, ingress, or both). 
+
+#### Default deny/allow behavior
+
+**Default allow** means all connections are allowed by default, unless otherwise specified. 
+**Default deny** means all connections are denied by default, unless explicitly allowed. 
```

Comment:
FYI you've put these on 2 separate lines, but they will appear on the same line when rendered into html. If you'd like them to appear on separate lines, put an additional newline between them.